test/unit/core/url-beautifier/factory.ts

```diff
@@ -111,6 +112,9 @@ suite('BeautifierFactory', ({ expect, stub }) => {
       beforeEach(() => {
         generator = new SearchGenerator.default(URL_BEAUTIFIER);
         parser = new SearchParser.default(URL_BEAUTIFIER);
+        Adapters.Configuration = <any>{
```

Comment:
If we do this we actually modify the global state, however we can stub `Adapters.Configuration.initialState()` to return data and it should continue working as expected